sdk/spring/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 5.19.0-beta.1 (Unreleased)
+
+### Spring Cloud Azure Autoconfigure
+This section includes changes in `spring-cloud-azure-autoconfigure` module.
+
+#### Bugs Fixed
+- Fix to registry Event Hubs connection string provider only when conn string is not empty. [#42919](https://github.com/Azure/azure-sdk-for-java/issues/42880).
```

Comment:
`Fix to registry Event Hubs connection string provider only when conn string is not empty.` -> `Fix error: Event Hubs connection string is still used when it's configured to empty string`